### ISSUE TITLE
feat: localization

### ISSUE THREAD

**Issue by Ushie** [ReVanced/revanced-manager#1467]
### Feature description

- [ ] Upload source file to Crowdin
- [ ] Support localization in ReVanced Manager
- [ ] Support Android 14's App Language switcher

### Motivation

ReVanced Manager should support localization, to make it more accessible

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by Ushie** [comment]
I can work on this after #1464 and #1465

**Comment by Ushie** [comment]
Done in df31b39cc8c1fbf00bc3301468e8e7e4b283caf2
